.github/workflows/ci.yml

```diff
@@ -93,6 +93,7 @@ jobs:
       matrix:
         os: [Ubuntu, MacOS]
         python:
+          - 3.6.0
```

Comment:
I like the idea here of pinning to 3.6.0 to make sure we don't regress, but doing so will (if I'm reading [the docs](https://github.com/actions/setup-python#available-versions-of-python) correctly) result in 3.6.0 getting downloaded, rather than using the preinstalled version of 3.6. That will slow down our CI a bit, and we have an ongoing fight to *speed up* our CI. @pypa/pip-committers awhat do you think? Is this worthwhile?

Also, we really don't need both 3.6.0 and 3.6...

Reply:
More than happy to remove this if you're comfortable. I added it as a way to demonstrate that this patch resolves the issue, but perhaps the value is outweighed by the interest in speeding up CI.

Reply:
I like the idea in principle, but let's see what the other committers think.

Reply:
You could have scheduled pipelines where you enable such configuration(s).. ?